assets/dev/js/editor/elements/views/behaviors/context-menu.js

```diff
@@ -55,6 +62,8 @@ module.exports = Marionette.Behavior.extend( {
 			} );
 		}
 
+		contextMenuGroups = elementor.hooks.applyFilters( 'elements/widget/onInitContextMenuGroups', contextMenuGroups, this );
```

Comment:
It shouldn't be here, this hook is related to widgets, and it's not a widget.
Plus, it won't work in older Pro versions if you add a new hook

Reply:
Changed.